nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -6,17 +6,19 @@ let
   cfg = config.services.nextcloud;
   fpm = config.services.phpfpm.pools.nextcloud;
 
-  phpPackage =
-    let
-      base = pkgs.php74;
-    in
-      base.buildEnv {
-        extensions = { enabled, all }: with all;
-          enabled ++ [
-            apcu redis memcached imagick
-          ];
-        extraConfig = phpOptionsStr;
-      };
+  phpPackage = pkgs.php74.buildEnv {
+    extensions = { enabled, all }:
+      enabled
+      ++ (with all;
+        [ imagick ] # Always enabled
```

Comment:
This doesn't even evaluate since there's a missing `++`.

It may be better to put `++` before a line, then it's also easier to comment a single line out during debugging.

Reply:
Better?